tide/wall/RenderController.h

```diff
@@ -82,6 +83,7 @@ public slots:
     SwapSyncObject<InactivityTimerPtr> _syncInactivityTimer;
     SwapSyncObject<OptionsPtr> _syncOptions;
     SwapSyncObject<MarkersPtr> _syncMarkers;
+    SwapSyncObject<ScreenLockPtr> _syncLock;
```

Comment:
please make sure that all the methods associated with syncLock are in the same order wrt. the others objects